src/logging/logging.ts

```diff
@@ -43,20 +35,45 @@ export class Log {
   constructor(private tag_:string) {
     this.logger = freedom.core().getLogger(this.tag_);
   }
+
+  private log_ = (level :string, args :any[]) :void => {
+    var message :string;
+
+    if (2 === args.length && 'string' === typeof(args[0]) && Array.isArray(args[1])) {
```

Comment:
I completely agree and normally do it that way, I was trying to follow https://docs.google.com/a/google.com/document/d/1ZV70oTNWO00bs0RcMwqr3FvKJXA6smYQ2kZ8Zr8YZnw/edit#heading=h.a3wjgdv24nv9, is that no longer accurate?